addons/crm/static/src/views/forecast_kanban/forecast_kanban_renderer.xml

```diff
@@ -3,13 +3,26 @@
     <t t-name="crm.ForecastKanbanRenderer" t-inherit="crm.CrmKanbanRenderer" t-inherit-mode="primary" owl="1">
         <KanbanColumnQuickCreate position="replace">
             <t t-if="isGroupedByForecastField()">
-                <ForecastKanbanColumnQuickCreate
-                    folded="true"
-                    onFoldChange="() => {}"
-                    onValidate.bind="addForecastColumn"
-                    exampleData="exampleData"
-                    groupByFieldString="getForecastQuickCreateTitle()"
-                />
+                <t t-if="canApplyExampleData()">
```

Comment:
to avoid the duplicated block, tried 
```suggestion
                <t t-set="canApplyExampleData" t-value="canApplyExampleData()">
                ...
                canApplyExampleData="canApplyExampleData"
                
```but go `canApplyExampleData`undefined, do you have any idea

Reply:
Not sure exactly what you mean, but there's currently an issue with sample data when grouped by date (as is the case in forecast data) so if you're getting something within the lines of "cannot read property date_deadline:month", that's unrelated to this. There's a fix coming soon :tm:

Reply:
Thanks @thiryr  for your comments, i'm checking